src/variants/registry/types.mligo

```diff
@@ -8,8 +8,8 @@
 type registry_key = string
 type registry_value = string
 
-type registry = (registry_key, registry_value) map
-type registry_affected = (registry_key, proposal_key) map
+type registry = (registry_key, registry_value) big_map
+type registry_affected = (registry_key, proposal_key) big_map
```

Comment:
It seems that this should be updated in the `docs/registry.md` file

Reply:
Have changed.